### PR TITLE
Fix build warnings

### PR DESCRIPTION
**Pull request by mueller-ma**
Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/model/Sitemap.kt [hunk 1]
```diff
@@ -38,14 +38,12 @@ fun Node.toSitemap(): Sitemap? {
     var label: String? = null
     var name: String? = null
     var icon: String? = null
-    var link: String? = null
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/model/Sitemap.kt]: It looks like the actual issue is that the 3rd parameter to the Sitemap construction below should be `link`, not `homepageLink`.

Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/model/Sitemap.kt]: The link property of `Sitemap` is only used in `SitemapTest#testGetLink()`. Should I remove `link` at all?

**Comment by mueller-ma** [comment]
There's also a warning, that `actualValue` (`ParsedState` Line 162) cannot be written to parcel, but `NumberState` is annotated with `@Parcelize`. I haven't found the usage of any parcel related functions of `ParsedState`, `Item` or `Widget`. Can `@Parcelize` be removed from these classes?

File: mobile/src/main/java/org/openhab/habdroid/model/Sitemap.kt (hunk 1 above)
Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/model/Sitemap.kt]: I don't have a strong opinion there ... do whatever you prefer there :-)

**Comment by maniac103** [comment]
> There's also a warning, that actualValue (ParsedState Line 162) cannot be written to parcel, but NumberState is annotated with @Parcelize. 

Probably need to implement `actualValue` as method instead of property then.

> I haven't found the usage of any parcel related functions of ParsedState, Item or Widget. Can @Parcelize be removed from these classes?

Well, I made them parcelable so they can be passed around as intent extras. It seems we're currently using this capability only for `ServerProperties`, but I'd keep the capability nevertheless.

**Pull request merged**